scheds/rust/scx_layered/src/bpf/intf.h

```diff
@@ -181,6 +181,7 @@ struct layer {
 	int			growth_algo;
 
 	u64			vtime_now;
+	u64			vtime_decay;
```

Comment:
This might be okay for now but `vtime_now` and `vtime_decay` are likely to make this cacheline very hot which can become a scalability problem for layers e.g. spanning multiple nodes or otherwise a large number of CPUs.

Ideally, everything that can be split per-DSQ should be per-DSQ. There's no reason to keep layer-wide `vtime_now`. This can be per-DSQ as long as delta translation is applied whenever tasks move from one DSQ to another.

As for `vtime_decay`, it's a bit trickier as it is something which needs to track global usage. However, this can also be kept per-DSQ and summed up periodically.